BB-SC/MWLayer/Platform-Health-Management/BB_Distributed_Health_Management.md

```diff
@@ -0,0 +1,83 @@
+
+# Distributed Health Management
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Platform Health Management
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
```

Comment:
e.g. https://opentelemetry.io - probably a lot out there

Reply:
Agreed. Will be taken care of in the next pull request.